packages/manager/modules/exchange/src/information/information.html

```diff
@@ -308,4 +308,18 @@
             </oui-tile>
         </div>
     </div>
+    <div class="row">
+        <!-- Advices -->
+        <div class="col-md-9">
+            <div class="row d-md-flex">
+                <div class="col-sm-6">
+                    <ovh-advices
+                        service-type="exchange"
+                        service-name="{{ ::ctrl.exchange.domain }}"
+                    >
+                    </ovh-advices>
+                </div>
+            </div>
+        </div>
+    </div>
```

Comment:
```suggestion
    <div class="row">
                <div class="col-sm-6">
                    <ovh-advices
                        service-type="exchange"
                        service-name="{{ ::ctrl.exchange.domain }}"
                    >
                    </ovh-advices>
                </div>
    </div>
```
?

Reply:
Exchange dashboard has col-9 and col-3 layout. Inside col-9, it again has col-6 and col-6 layout. To align to the first column properly, I had to use the same layout.